.github/workflows/tests.yml

```diff
@@ -37,26 +37,30 @@ jobs:
       - name: Lint ✨
         run: npm run test:lint
 
-  Build:
+  Release:
     needs: [Test, Lint]
-    if: github.ref == 'refs/heads/master'
+    if: |
+      github.ref == 'refs/heads/master' &&
+      github.event.repository.fork == false
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Install Dependencies
         run: npm ci
       - name: Build 🗜️
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           npm run build
           if ! git diff --quiet; then
             git config --global user.email "<>"
             git config --global user.name "MarkedJS bot"
             git commit -am "🗜️ build [skip ci]"
-            git push "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/markedjs/marked.git" HEAD:master
```

Comment:
Will semantic release push back the changes to the master branch?

Reply:
yes

Reply:
That is what @semantic-release/git does